.github/workflows/rust.yml

```diff
@@ -38,64 +38,34 @@ jobs:
         uses: Swatinem/rust-cache@v2
 
       - name: Rustfmt
-        uses: actions-rs/cargo@v1
-        with:
-          command: fmt
-          args: --all -- --check
```

Comment:
Looks like something we should fix in https://github.com/rerun-io/rerun_template/

Reply:
@jleibs already did so. But there we can use pixi